package.json

```diff
@@ -12,14 +12,13 @@
     "format": "prettier . --write --cache --cache-location=node_modules/.cache/.prettiercache --log-level=warn",
     "format:check": "prettier . --check --cache --cache-location=node_modules/.cache/.prettiercache",
     "lint": "turbo run lint",
-    "publint": "turbo run publint --concurrency=1",
     "are-the-types-wrong": "turbo run are-the-types-wrong",
     "manypkg": "manypkg check",
     "typesync": "typesync --dry=fail",
     "typecheck": "turbo run typecheck",
-    "knip": "knip",
+    "knip": "knip --treat-config-hints-as-errors",
```

Comment:
Just slipping this in (I requested this in https://github.com/webpro-nl/knip/issues/891).

Reply:
i would have preferred this as a config option (in the config file, i mean), but it's fine either way.

Reply:
Agreed, see https://github.com/webpro-nl/knip/issues/1026.

Reply:
Ha, and then he added it :)

Reply:
we will wait for him to add this as a config option in the next release. I prefer it.

Reply:
I filed #397, I'll revert this from this PR.